stock_picking_batch_extended/migrations/18.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,16 @@
+#  Copyright 2025 Camptocamp SA
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
```

Comment:
This migration script is needed to handle the return to an existent field `description_picking` from custom `description_warehouse`
It was requested in TODO